bridget/eddy/footprint.py

```diff
@@ -15,7 +17,242 @@
 
 """
 
-def footprint(t_air, a, p, u, cov_uw, cov_vw, cov_wt, var_v, z = 2.0, method = "Kormann_Meixner"):
+
+
+"""
+Functions to calculate meteorological paramters which are needed in the footprint function.
+"""
```

Comment:
put this in the module docstring at the first lines of the file. After that, no comments outside of functions anymore